<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCOverview</class>
 <widget class="QWidget" name="QCOverview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1503</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,8,2">
     <item>
      <widget class="CusTableView" name="QCProducets" native="true"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="8,2">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="TurnPageGridView" name="QCProjects" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="MachineM1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
border: 3px solid balck;
font: 26pt &quot;Microsoft YaHei UI&quot;;
}</string>
           </property>
           <property name="text">
            <string>M1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MachineM2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 26pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>M2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MachineM3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 26pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>M3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MachineM4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 26pt &quot;Microsoft YaHei UI&quot;;</string>
           </property>
           <property name="text">
            <string>M4</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="QCInfoArea" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(184, 184, 184);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="4,0,4,0,3,0,3,0,3">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="CusLineEditTopTitle" name="SampleID" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="CusLineEditTopTitle" name="SampleCode" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6" stretch="2,8">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
              </property>
              <property name="text">
               <string>试管架架号:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="RackNumberCBX">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>C01</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>C02</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>C03</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>C04</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>C05</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>C06</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>C07</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>C08</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>C09</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>C10</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,8">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
              </property>
              <property name="text">
               <string>试管架位置:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="RackIndexCBX">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="2,8">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 20pt &quot;Microsoft YaHei UI&quot;;</string>
              </property>
              <property name="text">
               <string>样本杯:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="SampleCupCBX">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>标准</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="QCApplyListBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}
</string>
       </property>
       <property name="text">
        <string>申请列表</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="QCSaveBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}
</string>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CusTableView</class>
   <extends>QWidget</extends>
   <header location="global">CusTableView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TurnPageGridView</class>
   <extends>QWidget</extends>
   <header location="global">TurnPageGridView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CusLineEditTopTitle</class>
   <extends>QWidget</extends>
   <header location="global">CusLineEditTopTitle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
